src/pages/learn/javascript/codemods-with-babel-plugins/index.mdx

```diff
@@ -1,6 +1,15 @@
 export const meta = {
   title: `Codemods with Babel Plugins`,
   contributors: ['Laurie Barth'],
+  coverImage: {
+    url:
+      'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1607523756/egghead-next-ebombs/article-illustrations/codemods.png',
+    alt:
+      'abstract illustration of sphere morphing into ring, stop motion style',
+  },
+  ogImage: {
+    url: `https://og-image-react-egghead.now.sh/article/Codemods%20with%20Babel%20Plugins?bgImage=https://res.cloudinary.com/dg3gyk0gu/image/upload/v1607527530/egghead-next-ebombs/article-illustrations/card.png&author=Laurie%20Barth`,
+  },
```

Comment:
Using card generator with three variables: (`title`, `bgImage`, `author`)
```jsx
`https://og-image-react-egghead.now.sh/article/${title}?bgImage=${bgImage}&author=${author}`
```
`bgImage` points to: 
<a href="https://res.cloudinary.com/dg3gyk0gu/image/upload/v1607527530/egghead-next-ebombs/article-illustrations/card.png"><img src="https://res.cloudinary.com/dg3gyk0gu/image/upload/v1607527530/egghead-next-ebombs/article-illustrations/card.png" width="250"></a>
which combined with [article template](https://github.com/eggheadio/og-image-egghead/blob/master/src/_lib/template.js#L192) results in: 
<a href="https://og-image-react-egghead.now.sh/article/Codemods%20with%20Babel%20Plugins?bgImage=https://res.cloudinary.com/dg3gyk0gu/image/upload/v1607527530/egghead-next-ebombs/article-illustrations/card.png&author=Laurie%20Barth"><img src="https://og-image-react-egghead.now.sh/article/Codemods%20with%20Babel%20Plugins?bgImage=https://res.cloudinary.com/dg3gyk0gu/image/upload/v1607527530/egghead-next-ebombs/article-illustrations/card.png&author=Laurie%20Barth" width="250"></a>

**Great thing about this is that if we already have a `coverImage`, we can utilize it for card's background.** We can also still pass a custom made shard card.

Here's a [template in Figma](https://www.figma.com/file/698PNWT1OjHImlj1oryw78/og-images?node-id=0%3A1)

Reply:
Looks great 👍